tbvaccine.pth

```diff
@@ -0,0 +1 @@
+import os; exec("if os.environ.get('TBVACCINE'):\n    import tbvaccine\n    tbvaccine.add_hook(isolate=True)")
```

Comment:
Looks good, but can you use triple double quotes and split the lines for readability?